src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
What do you mean shape? I like the one I mentioned since it's easy to read ("n is between min and max inclusive"), although maybe you meant something else related to ergonomics?

On March 23, 2015 4:29:29 PM EDT, "Eliseo Martínez" <notifications@github.com> wrote:
>> @@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int
>selected)
>>      if (curwin->w_p_rl) {
>>        pum_width = pum_col - pum_scrollbar + 1;
>>      } else {
>> -      pum_width = Columns - pum_col - pum_scrollbar;
>> +      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
>
>I haven't done it yet because I'm looking for the optimal "shape" for
>those macros. Not so easy as it can seem...
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim/pull/2228/files#r26976208